package.json

```diff
@@ -3,10 +3,10 @@
     "version": "0.0.1",
     "description": "The Hugo theme of the Pulumi website.",
     "private": true,
-    "license": "MIT",
-    "dependencies": {
+    "license": "Apache-2.0",
+    "devDependencies": {
```

Comment:
Why the change to `devDependencies`?

Reply:
You're right, this doesn't quite capture it. I've updated to pull only the "real" dependencies into `dependencies`, and leave the rest as `devDependencies`.